code/modules/surgery/operations/_operation.dm

```diff
@@ -454,6 +463,9 @@ GLOBAL_DATUM_INIT(operations, /datum/operation_holder, new)
 	/// Lasts muuuuuch longer.
 	var/datum/mood_event/surgery/surgery_failure_mood_event = /datum/mood_event/surgery/failure
 
+	/// Can this operation be performaed directly on a limb, or does it require a patient?
+	var/requires_patient = TRUE
```

Comment:
This should be an operation flag and inverted, ie `OPERATION_NO_PATIENT_REQUIRED` or smth